### ISSUE TITLE
Systematic way to convert multiple markdown files to html?

### ISSUE THREAD

**Issue by uncreativename** [atom/markdown-preview#289]
Is there an API to this package that allows converting multiple markdown files to html via command line or through atom's GUI interface?

Maybe something like this: `atom-markdown-preview ~/markdown_files --output ~/html_files`

**Comment by leipert** [comment]
Yep, have a look at http://pandoc.org/ or markdown-it https://github.com/markdown-it

It now depends, whether you want to merge the HTMl files or not:

```bash
# Single output files
for f in *.md ; do pandoc ${f} -f markdown -t html -s -o ${f}.html  ; done
# One output file
pandoc *.md -f markdown -t -s html -o merged.html
```
